<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayer</class>
 <widget class="QWidget" name="MusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Now Playing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Trebuchet MS'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Song Title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Artist</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Album</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLCDNumber" name="lcdTime"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnPrevious">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Playlist</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QTableWidget" name="tblMusic">
        <property name="columnCount">
         <number>2</number>
        </property>
        <column>
         <property name="text">
          <string>Artist</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnShuffle">
        <property name="text">
         <string>Shuffle</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>MusicPlayer</receiver>
   <slot>clearPlaylist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShuffle</sender>
   <signal>clicked()</signal>
   <receiver>MusicPlayer</receiver>
   <slot>shufflePlaylist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clearPlaylist()</slot>
  <slot>shufflePlaylist()</slot>
 </slots>
</ui>
